cudamapper/tests/Test_CudamapperIndexDescriptor.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+* Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include "gtest/gtest.h"
+
+#include "../src/index_descriptor.hpp"
+
+#include "cudamapper_file_location.hpp"
+
+namespace claragenomics
+{
+namespace cudamapper
+{
+
+// *** test IndexDescriptor and IndexDescriptorHash ***
+
+TEST(TestCudamapperIndexDescriptor, test_index_descriptor_getters)
+{
+    const read_id_t first_read      = 15;
+    const read_id_t number_of_reads = 156;
```

Comment:
I guess that should be a `number_of_reads_t` now. :)

Reply:
Yes.
There are also many other places which will benefit from this, leaving those as they are for now.
Done